ui/src/modals/newtask.vue

```diff
@@ -358,10 +378,17 @@ export default {
                 return match;
             });
 
-            let popular_ordered = this.apps.filtered.map(a=>{
-                if(!a.stats) a.stats = {users: 0};
+            //should i put a check for if filtered contains apps that are incompatible?
```

Comment:
when with questions, you can add comments here via github, it is easier to track than code comments